.rubocop_todo.yml

```diff
@@ -9,7 +9,7 @@
 # Offense count: 2
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 114
+  Max: 140
```

Comment:
This is global and because it’s the TODO file it will get accidentally deleted or removed. Instead disable a class locally with #rubocop/disable Metrics/ClassLength